### ISSUE TITLE
/ad only available to servers below a player limit

### ISSUE THREAD

**Issue by Wryz** [Minehut/Meta#897]
### Feature request

**Checklist**
- [Y] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [Y] This is not a bug report, support request or plugin request
- [Y] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [Y] This issue does not contain any sensitive information 

**Platform**
Feature for the Minehut lobby. Java & Bedrock

**Describe the feature**
We can agree that advertisements in lobbies have been getting out of hand. Most advertisements seem to be coming from top servers, drowning out smaller servers trying to get their server out there, and drowning out people just speaking in chat. To combat these issues, I believe the best solution is to limit the /ad command to servers that are BELOW a certain player limit.

**How the feature is useful**
I'm not alone when I say that I'm tired of seeing the same servers advertising 24/7. Maybe this also indirectly prevents the advertisement botting issue? It would most certainly clear chat up, and it would give smaller servers more of a chance to become popular. This would become beneficial to *most* people; just not the top 10 who all have more than a hundred people at peak hours.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by Blocksnmore** [comment]
This punishes smaller/growing servers when they get a wave of players from a youtuber etc, I don’t think minehut will implement that as it negatively affects the large community aspect of minehut

**Comment by Wryz** [comment]
> This punishes smaller/growing servers when they get a wave of players from a youtuber etc, I don’t think minehut will implement that as it negatively affects the large community aspect of minehut

I disagree with the idea that this would punish small/growing servers at all. I left the limitation up for question because the amount the limit is set at would determine how much of the community this change would affect. I'm having a hard time understanding how this would be a negative influence on small/growing servers. Is allowing chat flooding by larger servers allowing the growth of smaller servers? This would DEFINITELY not affect the community aspect of Minehut, if anything it STRENGTHENS it. Let's say the limit is 50 people. Now you won't get the same ads from servers above 50 people in lobby spamming the chat, which seems to make up the vast majority of chat messages in my experience. 

Please feel free to disagree with me, but provide some reasoning to help me understand something I'm missing.

**Comment by Tringlle** [comment]
I think this actually makes sense. It would limit the lobby ad spam and allow smaller servers to become larger.

**Comment by aient** [comment]
I don't agree with this because unsuccessful servers shouldn't be rewarded.

**Comment by PartyProNL** [comment]
@aient Just because a server has a lower player count doesn't mean it cannot be successful. Most big servers right now started out with just a few players.

I personally like this idea, although you could maybe change the player limit to whether the server is on the front page or not. The front page is basically one big advertisement, and those servers simply need less advertising to get players.

**Comment by aient** [comment]
> @aient Just because a server has a lower player count doesn't mean it cannot be successful. Most big servers right now started out with just a few players.
> 
> I personally like this idea, although you could maybe change the player limit to whether the server is on the front page or not. The front page is basically one big advertisement, and those servers simply need less advertising to get players.

I would say that a server with low player count and that isn't gaining much traction would be considered an unsuccessful server and there are more ways to prevent mass advertising coming soon so I just don't believe this idea is plausible.

**Comment by ntrogen** [comment]
Half of Minehut's smaller servers are complete trash anyway putting it bluntly, they're all little survival worlds with 0 effort put into them.

**Comment by TitanDevX** [comment]
This could be a good idea if they make the limit dynamic so it changes depends on the average player count on servers because its different from time to time

i like the idea

**Comment by NinjaTalonYT** [comment]
I like this idea, possibly for just the top row on the server list.

Whilst this punishes the highest servers on Minehut it allows for a sense of growth when it comes to smaller servers. The entire point of ads is to be able to grow a server, when the top servers are the only ones getting noticed in chat, that doesn't allow for other server growth. Ads are meant to allow players to grab an audience and show off their creations. I personally think limiting the top row of servers off of this could be a good idea.

This may be a controversial change but it could definitely allow for more server growth all around the network and would down the amount of ad-bots (which seem correlated with the top servers).
